python/snewpy/test/simplerate_integrationtest.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""Integration test based on SNEWS2.0_rate_table_singleexample.py
+"""
+import unittest
+import os
+from snewpy import snowglobes
+
+
+class TestSimpleRate(unittest.TestCase):
+
+    def test_simplerate(self):
+        """Integration test based on SNEWS2.0_rate_table_singleexample.py
+        """
+        # Hardcoded paths on GitHub Action runner machines
+        SNOwGLoBES_path = os.environ['SNOWGLOBES']
+        SNEWPY_model_dir = "models/"
+
+        distance = 10  # Supernova distance in kpc
+        detector = "wc100kt30prct" #SNOwGLoBES detector for water Cerenkov
+        modeltype = 'Bollig_2016' # Model type from snewpy.models
+        model = 's11.2c' # Name of model
+        transformation = 'AdiabaticMSW_NMO' # Desired flavor transformation
+
+        # Construct file system path of model file and name of output file
+        model_path = SNEWPY_model_dir + "/" + modeltype + "/" + model
+        outfile = modeltype + "_" + model + "_" + transformation
+
+        # Now, do the main work:
+        print("Generating fluence files ...")
+        tarredfile = snowglobes.generate_fluence(model_path, modeltype, transformation, distance, outfile)
+
+        print("Simulating detector effects with SNOwGLoBES ...")
+        snowglobes.simulate(SNOwGLoBES_path, tarredfile, detector_input=detector, detector_effects=False)
+
+        print("Collating results ...")
+        tables = snowglobes.collate(SNOwGLoBES_path, tarredfile, detector_input=detector, skip_plots=True, smearing=False)
+
+        # Use results to print the number of events in different interaction channels
+        key = f"Collated_{outfile}_{detector}_events_smeared_weighted.dat"
```

Comment:
Use unsmeared events for `SimpleRate` test
```suggestion
        key = f"Collated_{outfile}_{detector}_events_unsmeared_weighted.dat"
```